sycl/test-e2e/Graph/graph_exception_global_device_extension.cpp

```diff
@@ -144,6 +146,14 @@ template <OperationPath PathKind> void test() {
 }
 
 int main() {
+  {
+    queue queue;
```

Comment:
Can we call this `Queue` and pass it to `test()` as a parameter, rather than creating a new queue in `test()`

Reply:
Updated thanks, made this consistent across all the tests.